mimic/test/test_nova.py

```diff
@@ -940,6 +941,41 @@ def test_rebuild(self):
             }
         })
 
+    def test_create_image(self):
```

Comment:
We should really have test docstrings in the style of https://jml.io/pages/test-docstrings.html for all our unit tests.